latex/main.tex

```diff
@@ -2,27 +2,27 @@
 
 %% To include CJK character, one needs to use xelatex to compile
 %% and also to have a CJK font installed (current set to `Noto Sans CJK TC`)
-%% Otherwise, comment out the following two lines.  
+%% Otherwise, comment out the following two lines.
 %\usepackage{xeCJK}
 %\setCJKmainfont{Noto Sans CJK TC}
 
 %% basic font settings
-\usepackage[T1]{fontenc} 
+\usepackage[T1]{fontenc}
 \usepackage[default]{raleway}
 \providecommand{\raleway}{} % in case we are not using xeLaTex
 \newcommand*{\latinfont}{\raleway}
 \hyphenpenalty=10000
 
 %% essential packages
-\usepackage[boxed]{ticket} %when ready to print, remove the "boxed" option
-\usepackage[usenames]{xcolor}
+\usepackage{ticket} %when ready to print, remove the "boxed" option
```

Comment:
```suggestion
\usepackage{ticket} %to see the badge boundary (for testing purpose), add [boxed] as an option
```

Reply:
ooh yes, my bad, I removed that by accident. I think it might actually be better to keep the [boxed] by default